Dockerfile

```diff
@@ -15,6 +15,10 @@ ADD . .
 
 WORKDIR /root
 
+# Instructs distillery to interpolate environment variables at runtime.
```

Comment:
You don't need this here: it's already on line 31. This environment is used at compile time, so this runtime variable doesn't have an effect.

Reply:
Removed 👍